.github/workflows/pr-deploy-cleanup.yaml

```diff
@@ -0,0 +1,63 @@
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  cleanup:
+    name: Cleanup merged deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check user permission
+        id: check
+        uses: scherermichael-oss/action-has-permission@17f29510f1bf987b916c8cbb451566a56eed23f1
```

Comment:
Can we check if the user has permissions in some other way? Asking as we are passing a GitHub token to a GitHub Action that doesn't seems to be maintained (latest release is from 2020) and it's not from a well known maintainer (GitHub, Google, etc..)

Reply:
Sure, can swap it out for some javascript I think! I did skim the code of the action + pinned to the SHA, but totally get the concern